cotel/src/main/scala/day4/HighEntropyPassphrasses.scala

```diff
@@ -0,0 +1,35 @@
+package day4
+
+import scala.io.Source
+
+object HighEntropyPassphrasses {
+
+  def isValidPassphrase(passphrase: String, checked: List[String] = List(), result: Boolean = true): Boolean = {
+    def recursion(passphrase: List[String], checked: List[String] = List(), result: Boolean = true): Boolean =
+      (passphrase, checked, result) match {
+        case (List(), _, acc) => acc
```

Comment:
I search for something like that in kotlin but :(

Reply:
I think Kategory allows contructs like this with lists 🤔